web/styles/compose.css

```diff
@@ -16,7 +16,7 @@
             & span {
                 font-size: 1.2em !important;
                 font-weight: 400;
-                line-height: 1em;
+                line-height: 20px;
```

Comment:
Nice work, @vighneshbhat9945. That's an especially good catch here on the mobile compose button.

Two more things:

1. I should've thought of this ahead of time, but let's go ahead and create a CSS variable called something like `--line-height-compose-buttons` in `zulip.css` (study the stylesheet for an optimal place to include that), and then reference it from these three areas in your changes.
2. Update your screenshots with the final changes in place; it would be good to capture a mobile-width screen too, given that change now reflected here.